Makefile

```
@@ -67,7 +67,8 @@ pre-requirements: ## install Python requirements for running pip-tools
 	pip install -qr requirements/edx/pip-tools.txt
 
 requirements: pre-requirements ## install development environment requirements
-	pip-sync -q requirements/edx/development.txt requirements/edx/private.*
+	# Wildcard "txt*" is so private.txt is included only if it exists
```

You may want to tweak this to reflect the changes (there's no more `txt*`), but otherwise LGTM.

```suggestion
	# Wildcard is so private.txt is included only if it exists
```